cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -278,3 +278,36 @@ def test_vmware_provisioned_vm_host_relationship():
             3.CFME Provisioned VM should have host relationship.
     """
     pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+def test_esxi_reboot_not_orphan_vms():
+    """
+    By mimicking ESXi reboot effect on VMs in CFME, make sure they are not getting marked orphaned.
+
+    Bugzilla:
+        1695008
+
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Infra
+        caseimportance: critical
+        initialEstimate: 1/2h
+        testtype: integration
+        testSteps:
+            1.Add VMware provider to CFME
+            2.SSH to CFME appliance and perform following steps in rails console
+                '''
+                ems = ManageIQ::Providers::Vmware::InfraManager.find_by(:name => "name of your vc")
+                vm = ems.vms.last # Or do vms[index] and find a vm to work with
+                puts "ID [#{vm.id}] ems_ref [#{vm.ems_ref}]"
+                vm.update_attributes(:uid_ems => SecureRandom.uuid)
+                '''
+            3.Refresh the provider
+        expectedResults:
+            1.Provider added successfully and is refreshed
+            2.VM's uid_ems is modified
+            3.VM is still active and usable in cloudforms, not archived/orphaned.
+    """
+    pass
```

Comment:
when you have docstring, you don't need to put pass.

Reply:
^ This was discussed in the reviewers call, for now we are keeping `pass`.